Makefile

```diff
@@ -45,7 +45,7 @@ NODE_CONTAINER_NAME?=calico/node
 NODE_CONTAINER_FILES=$(shell find $(NODE_CONTAINER_DIR)/filesystem -type f)
 NODE_CONTAINER_CREATED=$(NODE_CONTAINER_DIR)/.calico_node.created
 NODE_CONTAINER_BIN_DIR=$(NODE_CONTAINER_DIR)/filesystem/bin
-NODE_CONTAINER_BINARIES=startup startup-go allocate-ipip-addr calico-felix bird calico-bgp-daemon confd libnetwork-plugin
+NODE_CONTAINER_BINARIES=startup startup-go allocate-ipip-addr allocateIPIPAddr calico-felix bird calico-bgp-daemon confd libnetwork-plugin
```

Comment:
We already have the allocate-ipip-addr make target/binary.  I think we should just use that rather than adding another allocateIPIPAddr